### ISSUE TITLE
GNOME Software says PySolFC is unsafe

### ISSUE THREAD

**Issue by KristjanESPERANTO** [shlomif/PySolFC#356]
Switching to another Windowing System is probably not that easy, is it?

You use Tkinter, don't you? I wouldn't have thought that would be classified as legacy.

![Screenshot from 2024-01-07 14-37-57](https://github.com/shlomif/PySolFC/assets/35647502/679acfb4-1f0f-404c-8c08-e8db29565814)

**Comment by joeraz** [comment]
Yes, PySolFC uses Tkinter, and it would be very difficult to change.  In fact, I don't think it would be practical unless we were able to get a few more developers involved.  There is a beta of a gtk-based window system in the code, but I don't know enough about the status of that.

That being said, this error looks like it's coming from flathub, where there are reports of this being not due to tkinter, but the fact that the win system is x11.  From what I hear, changing this to x11-fallback should get rid of the issue, which I intend to test out with the next release.

**Comment by KristjanESPERANTO** [comment]
Ah, by windowing system they mean X11. I obviously wrongly pointed the finger at Tkinter. I had a look at [the flatpak options](https://docs.flatpak.org/en/latest/flatpak-command-reference.html), `--nosocket=fallback-x11` sounds promising. Apart from using flatpak, I have never looked deeper into it. I am curious to see if it works :slightly_smiling_face:
